chart/templates/webserver/webserver-ingress.yaml

```
@@ -31,7 +31,7 @@ metadata:
     heritage: {{ .Release.Service }}
   {{- with .Values.ingress.web.annotations }}
   annotations:
-  {{ toYaml . | indent 4 }}
+{{ toYaml . | indent 4}}
```

```suggestion
  {{- toYaml . | nindent 4 }}
```
How about using nindent for readability?

---

Using `-` after opening braces makes sense as it removes preceding whitespaces during rendering, but I think nindent doesn't give much meaning here.